pkgs/build-stylus-settings/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchurl,
+}:
+let
+  stylusSectionsUtil = fetchurl {
+    url = "https://raw.githubusercontent.com/openstyles/stylus/8fe35a4b90d85fb911bd7aa1deab4e4733c31150/src/js/sections-util.js";
+    hash = "sha256-OH4WStQLx2BI3MqvzmoDKsdFYW5/wp8Yc9faM/VzlQI=";
+  };
+
+  usercssMetaTypes = fetchurl {
+    url = "https://raw.githubusercontent.com/catppuccin/userstyles/714b153c7022c362a37ab8530286a87e4484a828/scripts/types/usercss-meta.d.ts";
+    hash = "sha256-e/gazaMzr0WcarsE/NqCVT/iCG9CoB0lnZiOr1l19Jk=";
+  };
+in
+buildNpmPackage (finalAttrs: {
+  pname = "build-stylus-settings";
+  version = "0.2.0";
+
+  src = builtins.path {
+    path = ./src;
+    name = finalAttrs.pname + "-src";
+  };
```

This is better imo https://noogle.dev/f/lib/cleanSource
```suggestion
  src = lib.cleanSource ./src;
```